pkg/descheduler/controllers/migration/arbitrator/arbitrator.go

```diff
@@ -167,7 +202,7 @@ func (a *arbitratorImpl) doOnceArbitrate() {
 	// filter
 	for _, job := range jobs {
 		pod := podOfJob[job]
-		isFailed, isPassed := a.filter(pod)
+		isFailed, isPassed := a.filtering(pod)
```

Comment:
Although if the job has passed the filter and updates the quorum status of the job that has passed, the status of the job notifier is still old and therefore when the next job is processed, it should fail but pass, this possibility exists.